scripts/send_file.py

```diff
@@ -0,0 +1,48 @@
+import json
+import requests
+import time
+
+# How many times would you like to run? More times = more of the same data
+# Use loops to continuously start over and feed the data in if you want to test longer
+loops = 1
+
+# Desired chunk size, 1000 events would be standard
+chunk, chunksize = [], 1000
+
+# Data Reduction account: Weblog 200 Reduce pipeline
+api_key = "<INSERT KEY HERE>"
+api_url = "<YOUR PIPELINE URL>"
+my_file = "<YOUR LOCAL FILE>"
+
+# Set the timer between requests in seconds, use 0 for none
+s = 1
+
+# For sending the data chunks to the pipeline HTTP endpoint
+def use_requests(chunk):
+    response = requests.post(api_url, headers={'Authorization': api_key}, data=json.dumps(chunk))
+    json_response = json.loads(response.text)
+    print(json_response)
+    time.sleep(s) # set how long to wait between requests
+    return
+
+def open_file(filename):
+    reader = open(filename, mode='r')
+    for i, line in enumerate(reader):
+        if (i % chunksize == 0 and i > 0):
```

Comment:
Minor, but what if they hit the max 2MB size in the chunk?

Reply:
Ah good point. I can add a check for the size.

Reply:
Okay, I fixed this by counting the string size as it increments. I also noticed a bug in that I was dropping the lines that happened during the request execution.